app_customers/lib/routes/app_router_gorouter.dart

```diff
@@ -80,6 +96,7 @@ class _GoRouterRoutesProvider {
               buyerGatewayId: buyerGatewayId,
               featureReference: featureReference,
               receiverOperator: receiverOperator,
+              forfeitId: forfeitId == 'null' ? null : forfeitId,
```

Comment:
can we avoid to put value 'null' ??

Reply:
nope when there is element depend on request we retrieve 'null' or null so it is very important  to have the 2 verification